ui/src/app/tasks/task-apps/task-apps.component.ts

```diff
@@ -0,0 +1,60 @@
+import { Component, OnInit } from '@angular/core';
+import { Subscription } from 'rxjs/Subscription';
+import { ToastyService, ToastyConfig, ToastOptions, ToastData } from 'ng2-toasty';
+import { Router, ActivatedRoute, Params } from '@angular/router';
+import { AppRegistration } from '../../apps/model/app-registration';
+import { Page } from '../../shared/model/page';
+import { AppsService } from '../../apps/apps.service';
```

Comment:
@jvalkeal I can do the refactoring during merge but I want to wait until @cppwfs merges my other PR as that is apps specific. As I do some state-management in the app service, I need to be a bit careful refactoring it. Most likely delegating to the *shared (core) apps service that will be stateless*.